src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/EnrichedProbationSearchResults.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation
+
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.ProbationSearchResult
+
+data class EnrichedProbationSearchResults(
+  val results: List<ProbationSearchResult>,
+  val inPrisonCount: Int,
+  val onProbationCount: Int,
+)
```

Comment:
is this being returned to the client? If so I think it should live in model and have swagger annotations?

Reply:
Good spot - I also forgot to update the swagger doc on the endpoint so will also do that.

Reply:
Done